lagom1-javadsl-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/ConductRServiceLocator.scala

```diff
@@ -20,7 +22,7 @@ import com.typesafe.conductr.bundlelib.play.api.LocationService
 /**
  * ConductRServiceLocator implements Lagom's ServiceLocator by using the ConductR Service Locator.
  */
-class ConductRServiceLocator @Inject() (locationService: LocationService, cache: CacheLike) extends ServiceLocator {
+class ConductRServiceLocator @Inject() (locationService: LocationService, cache: CacheLike, circuitBreakers: CircuitBreakers) extends CircuitBreakingServiceLocator(circuitBreakers) {
```

Comment:
If we are to support circuit breaking in ConductR, then the ConductR service locator needs to either implement its own circuit breaking logic, or extend Lagom's CircuitBreakingServiceLocator.